api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/CasConfigurationProperties.java

```diff
@@ -242,6 +243,11 @@
     @NestedConfigurationProperty
     private WarningCookieProperties warningCookie = new WarningCookieProperties();
 
+    /**
+     * Distributed OAuth Session cookie.
```

Comment:
This is not entirely accurate, as the distributed-session store is not exclusively used for OAuth/OIDC but also delegated authn.